reference/v8.json

```diff
@@ -1241,43 +1359,64 @@
   },
   "filter_operator": {
     "type": "enum",
-    "values": [
-      "==",
-      "!=",
-      ">",
-      ">=",
-      "<",
-      "<=",
-      "in",
-      "!in",
-      "all",
-      "any",
-      "none",
-      "has",
-      "!has"
-    ],
+    "values": {
+      "==": {
+          "doc": "The equal-to operator."
```

Comment:
Let's pull in the descriptions from https://www.mapbox.com/mapbox-gl-style-spec/#types-filter for these.